src/Domain/Model/Conversation/Conversation.php

```diff
@@ -5,15 +5,18 @@
 use Chatbot\Domain\Event\ConversationCreated;
 use Chatbot\Domain\Event\PairAdded;
 use Chatbot\Domain\EventFacade\EventFacade;
+use Chatbot\Domain\Model\Context\ContextId;
 use DateTimeImmutable;
 use Safe\DateTimeImmutable as SafeDateTimeImmutable;
 
 class Conversation
 {
     public function __construct(
         private PairArray $pairs,
+        private ContextId $context,
```

Comment:
Erreur phpstan: Property Chatbot\Domain\Model\Conversation\Conversation::$context is never read, only written.